article.ipynb

```diff
@@ -136,7 +136,7 @@
     "\n",
```

Comment:
<p>section (8.4) to replace by <strong>Optimisation and improvement of the model</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jdh-observer/ZDFmmD3xv4S7/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>